packer/templates/vagrant-box-atom/provisioning/host_vars/all

```diff
@@ -105,7 +105,8 @@ elasticsearch_network_host: "127.0.0.1"
 #
 # percona role
 #
-
+ansible_python_interpreter: "/usr/bin/python3"
+ansible_distribution_release: "focal"
```

Comment:
Will this be needed in all playbooks? Maybe we can find a way to do it in the roles to avoid that. In some roles we compare the Ubuntu version differently, could we use that method in https://github.com/artefactual-labs/ansible-nginx/pull/13/files to avoid this var? Would it be possible to install `python3-is-python` in 20.04 to avoid the other?

Reply:
Also, why are they part of the percona role section?

Reply:
Good idea @jraddaoui! Do you know of an example where the Ubuntu version is checked in one of the roles? I will look into 'python3-is-python'...

Reply:
https://github.com/artefactual-labs/ansible-atom/blob/master/tasks/fop.yml#L3-L17